.github/workflows/publish.yaml

```diff
@@ -58,8 +58,9 @@ jobs:
           [ ! -z "${{ secrets.ORG_GPG_PASSPHRASE }}" ] && 
           [ ! -z "${{ secrets.ORG_GPG_PRIVATE_KEY }}" ] && 
           [ ! -z "${{ secrets.ORG_OSSRH_USERNAME }}" ] && 
-          [ ! -z "${{ secrets.ORG_OSSRH_PASSWORD }}" ]  && 
-          echo "HAS_OSSRH=true" >> $GITHUB_OUTPUT
+          [ ! -z "${{ secrets.ORG_OSSRH_PASSWORD }}" ]  && echo "HAS_OSSRH=true" >> $GITHUB_OUTPUT
+          [ ! -z "${{ secrets.SWAGGERHUB_API_KEY }}" ]  && 
+          [ ! -z "${{ secrets.SWAGGERHUB_USER }}" ]  && 
```

Comment:
Pls add `echo "HAS_SWAGGER=true >> $GITHUB_OUTPUT` here.
Pls also add the `HAS_SWAGGER: ${{ steps.secret-presence.outputs.HAS_SWAGGER }}` to the job outputs (after L52), otherwise it won't be available